### ISSUE TITLE
File type and format of content 

### ISSUE THREAD

**Issue by wukong5777** [Cogito2012/UString#8]
Hi, thanks for your work.
When I run dataloader.py, extract_res101_dad.py and split_dad.py, I find the type and format of the file to load is not clear.  The file seems that the loading data is not the annotation file of DAD dataset. What is the format of "all_data",  and how is the content ['data']['labels']['det'] like?
Thanks in advance.

**Comment by Cogito2012** [comment]
@wukong5777 Thanks for your interest in this work!
For DAD dataset, the annotations are originally provided by [DAD Official](https://github.com/smallcorgi/Anticipating-Accidents) and they are grouped into batches (each file contains features and annotations of a batch of 10 files). However, it is more standard to split them into separate files to enable random shuffling during training. To this end, we use the script [`./script/split_dad.py`](https://github.com/Cogito2012/UString/blob/master/script/split_dad.py) to re-organize DAD into our defined data format. For each video sample, we store its annotations and features in a single npz file containing `data`, `det`, `labels`, and `ID`. The detailed information for each of these 4 items are the same as introduced in [CarCrashDataset ReadMe](https://github.com/Cogito2012/CarCrashDataset). 

After the `split_dad.py` is applied, the `dataloader.py` can be used in this repo. Besides, we attempted to use ResNet-101 features to replace the original vgg features from DAD so that `extract_res101_dad.py` is provided as a helpful tool.

Hope this clarification can help your work. Thanks!

**Comment by ZhangJianshu3** [comment]
Thanks for your work!
I have cloned your code and Pre-trained UString Model (final_model_crash_vgg16.pth). When setting the file 'final_model_crash_vgg16.pth' as the  model file and placed it at `demo/final_model_ccd.pth` and try to run the run_demo.sh, I find that the result is random (For the same video, the output of each command is inconsistent) and is not consistent with the given example.

Did I use the wrong model? The Pre-trained UString Model (final_model_crash_vgg16.pth) is not the final model and 
it should not be used identify in demo?
